packages/core/src/node/utils/normalizePath.ts

```diff
@@ -41,9 +41,9 @@ function absolutePathToRoutePrefix(
     ),
   );
   const routeService = RouteService.getInstance();
-  const [basePrefix, versionPrefix, langPrefix] =
+  const [_, versionPrefix, langPrefix] =
```

Comment:
```suggestion
  const [versionPrefix, langPrefix] =
```

If I didn't mis-read, there is no `base` now?